diff -burN stunnel-4.06.orig/src/client.c stunnel-4.06/src/client.c
--- stunnel-4.06.orig/src/client.c	2004-12-26 18:33:15.114067672 +0100
+++ stunnel-4.06/src/client.c	2004-12-26 19:11:56.030234784 +0100
@@ -705,8 +705,8 @@
     leave_critical_section(CRIT_INET);
 
     if (!result) {
-        s_log(LOG_WARNING, "Connection from %s REFUSED by libwrap",
-            c->accepting_address);
+        s_log(LOG_WARNING, "Connection to service %s from %s REFUSED by libwrap",
+            c->opt->servname, c->accepting_address);
         s_log(LOG_DEBUG, "See hosts_access(5) for details");
         return -1; /* FAILED */
     }
