diff -ur stunnel-4.12/configure.ac stunnel-4.12.new/configure.ac
--- stunnel-4.12/configure.ac	2005-09-29 19:56:55.000000000 +0000
+++ stunnel-4.12.new/configure.ac	2005-09-30 07:20:00.968968264 +0000
@@ -42,7 +42,7 @@
 # Checks for standard libraries
 #AC_CHECK_LIB(dl, dlopen)
 #AC_CHECK_LIB(dld, shl_load)
-AC_CHECK_LIB(nsl, gethostbyname)
+AC_CHECK_FUNC(gethostbyname, [], AC_CHECK_LIB(nsl, gethostbyname))
 AC_CHECK_LIB(socket, socket)
 AC_CHECK_LIB(util, openpty)
 AC_CHECK_LIB(z, inflateEnd)
