--- stunnel-3.8/Makefile.in.wiget	Tue Feb 22 13:08:16 2000
+++ stunnel-3.8/Makefile.in	Mon Feb 28 13:10:25 2000
@@ -2,13 +2,13 @@
 # Makefile for stunnel by Michal Trojnara 1998-2000 #
 #####################################################
 
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-sbindir=@sbindir@
-libdir=@libdir@
-man8dir=@mandir@/man8
-piddir=@localstatedir@/stunnel
-ssldir=@ssldir@
+prefix=$(DESTDIR)@prefix@
+exec_prefix=$(DESTDIR)@exec_prefix@
+sbindir=$(DESTDIR)@sbindir@
+libdir=$(DESTDIR)@libdir@
+man8dir=$(DESTDIR)@mandir@/man8
+piddir=$(DESTDIR)@localstatedir@/stunnel
+ssldir=$(DESTDIR)@ssldir@
 certdir=$(ssldir)/certs
 @SET_MAKE@
 VERSION=stunnel-@VERSION@
