--- stunnel-4.05/tools/stunnel.conf-sample.in.orig	2004-05-09 02:40:26.000000000 +0200
+++ stunnel-4.05/tools/stunnel.conf-sample.in	2004-05-09 02:42:32.000000000 +0200
@@ -2,12 +2,12 @@
 # Copyright by Michal Trojnara 2002
 
 # Comment it out on Win32
-cert = @prefix@/etc/stunnel/mail.pem
-chroot = @prefix@/var/run/stunnel/
+#cert = /etc/stunnel/mail.pem
+#chroot = @prefix@/var/run/stunnel/
 # PID is created inside chroot jail
-pid = /stunnel.pid
-setuid = nobody
-setgid = nogroup
+pid = /var/run/stunnel/stunnel.pid
+setuid = stunnel
+setgid = stunnel
 
 # Workaround for Eudora bug
 #options = DONT_INSERT_EMPTY_FRAGMENTS
@@ -33,26 +33,26 @@
 
 # Service-level configuration
 
-[pop3s]
-accept  = 995
-connect = 110
-
-[imaps]
-accept  = 993
-connect = 143
-
-[ssmtp]
-accept  = 465
-connect = 25
-
-[s1]
-accept  = 5000
-connect = mail.osw.pl:110
-# delay = yes
-
-[s2]
-accept  = 5001
-connect = mail.osw.pl:25
+#[pop3s]
+#accept  = 995
+#connect = 110
+
+#[imaps]
+#accept  = 993
+#connect = 143
+
+#[ssmtp]
+#accept  = 465
+#connect = 25
+
+#[s1]
+#accept  = 5000
+#connect = mail.osw.pl:110
+#delay = yes
+
+#[s2]
+#accept  = 5001
+#connect = mail.osw.pl:25
 
 #[https]
 #accept  = 443
