--- stunnel-4.04/src/Makefile.am.orig	Sun Jan 12 19:16:35 2003
+++ stunnel-4.04/src/Makefile.am	Sun Jan 12 19:17:30 2003
@@ -18,26 +18,8 @@
 libstunnel_la_SOURCES = $(shared_sources)
 libstunnel_la_LDFLAGS = -avoid-version
 
-# Win32 executable
-
-EXTRA_DIST = stunnel.exe make.bat Makefile.w32
-EXTRA_PROGRAMS = stunnel.exe
-stunnel_exe_SOURCES =  $(common_headers) $(common_sources) $(win32_sources)
-
-OPENSSLDIR=/usr/src/openssl-0.9.7
-WINCFLAGS=-O2 -Wall -I$(OPENSSLDIR)/outinc \
-	-DUSE_WIN32=1 -DHAVE_OPENSSL=1 -DVERSION=\"@VERSION@\"
-WINLIBS=-L$(OPENSSLDIR)/out -leay32 -lssl32 -lwsock32 -lgdi32 -mwindows
-WINOBJ=client.obj log.obj options.obj protocol.obj sselect.obj \
-    ssl.obj sthreads.obj stunnel.obj gui.obj resources.obj
-WINGCC=i586-mingw32msvc-gcc
-WINDRES=i586-mingw32msvc-windres
-
 SUFFIXES = .c .rc .obj
 
-stunnel.exe: $(WINOBJ)
-	$(WINGCC) -s -o stunnel.exe $(WINOBJ) $(WINLIBS)
-
 .c.obj: $(common_headers)
 	$(WINGCC) $(WINCFLAGS) -c $< -o $@
 
--- stunnel-4.04/tools/Makefile.am	Sun Jan 12 05:22:12 2003
+++ stunnel-4.04/tools/Makefile.am.new	Sun Jan 12 19:22:57 2003
@@ -16,7 +16,7 @@
 		then dd if="$(RANDOM_FILE)" of=stunnel.rnd bs=256 count=1; \
 			RND="-rand stunnel.rnd"; \
 		else RND=""; fi; \
-	$(openssl) req -new -x509 -days 365 -nodes $$RND \
+	echo -e "\n\n\n\n\n\n" | $(openssl) req -new -x509 -days 365 -nodes $$RND \
 		-config $(srcdir)/stunnel.cnf -out stunnel.pem -keyout stunnel.pem; \
 	test $(USE_DH) -eq 0 || $(openssl) gendh $$RND 512 >> stunnel.pem
 	$(openssl) x509 -subject -dates -fingerprint -noout \
