--- stunnel-4.15/tools/stunnel.conf-sample.in.orig	2006-01-07 14:58:50.000000000 +0100
+++ stunnel-4.15/tools/stunnel.conf-sample.in	2006-04-01 23:31:04.987918000 +0200
@@ -3,15 +3,16 @@
 ; Please make sure you understand them (especially the effect of chroot jail)
 
 ; Certificate/key is needed in server mode and optional in client mode
-cert = @prefix@/etc/stunnel/mail.pem
-;key = @prefix@/etc/stunnel/mail.pem
+cert = /etc/stunnel/mail.pem
+;key = /etc/stunnel/mail.pem
 
 ; Some security enhancements for UNIX systems - comment them out on Win32
-chroot = @prefix@/var/lib/stunnel/
-setuid = nobody
-setgid = nogroup
+;chroot = @prefix@/var/lib/stunnel/
+setuid = stunnel
+setgid = stunnel
 ; PID is created inside chroot jail
-pid = /stunnel.pid
+;pid = /stunnel.pid
+pid = /var/run/stunnel/stunnel.pid
 
 ; Some performance tunings
 socket = l:TCP_NODELAY=1
@@ -43,17 +44,17 @@
 
 ; Service-level configuration
 
-[pop3s]
-accept  = 995
-connect = 110
-
-[imaps]
-accept  = 993
-connect = 143
-
-[ssmtp]
-accept  = 465
-connect = 25
+;[pop3s]
+;accept  = 995
+;connect = 110
+
+;[imaps]
+;accept  = 993
+;connect = 143
+
+;[ssmtp]
+;accept  = 465
+;connect = 25
 
 ;[https]
 ;accept  = 443
