--- stunnel-3.6/configure.in.wiget	Thu Feb  3 10:41:32 2000
+++ stunnel-3.6/configure.in	Thu Feb 10 04:32:32 2000
@@ -14,7 +14,7 @@
 AC_DEFINE_UNQUOTED(VERSION, "$VERSION")
 
 AC_MSG_CHECKING([for SSL config directory])
-for dir in /usr/local/openssl /usr/lib/openssl /var/openssl; do
+for dir in /usr/local/openssl /usr/lib/openssl /var/openssl /var/state/openssl; do
     sslcnf="$dir"
     if test -d "$dir/certs"; then
         break
