--- stunnel-5.14/src/client.c.orig	2015-04-04 08:02:22.369998614 +0200
+++ stunnel-5.14/src/client.c	2015-04-04 08:05:08.149995161 +0200
@@ -1039,8 +1039,8 @@
     while(*user==' ') /* skip leading spaces */
         ++user;
     if(strcmp(user, c->opt->username)) {
-        s_log(LOG_WARNING, "Connection from %s REFUSED by IDENT (user \"%s\")",
-            accepted_address, user);
+        s_log(LOG_WARNING, "Connection to service %s from %s REFUSED by IDENT (user \"%s\")",
+            c->opt->servname, accepted_address, user);
         str_free(line);
         longjmp(c->err, 1);
     }
