diff -durN stunnel-3.4a.orig/ssl.c stunnel-3.4a/ssl.c
--- stunnel-3.4a.orig/ssl.c	Mon Sep 13 11:00:20 1999
+++ stunnel-3.4a/ssl.c	Mon Sep 13 11:02:10 1999
@@ -52,7 +52,7 @@
 #else /* USE_WIN32 */
 
 /* directory for certificate */
-#define CERT_DIR ssldir "/certs"
+#define CERT_DIR "/var/state/openssl/certs"
 
 /* default certificate */
 #define DEFAULT_CERT CERT_DIR "/stunnel.pem"
@@ -61,7 +61,7 @@
 #define CA_DIR CERT_DIR "/mytrusted"
 
 /* certificate used for sign our client certs */
-#define CLIENT_CA ssldir "/bin/demoCA/cacert.pem"
+#define CLIENT_CA "/var/state/openssl/cacert.pem"
 
 #endif /* USE_WIN32 */
 
diff -durN stunnel-3.4a.orig/stunnel.c stunnel-3.4a/stunnel.c
--- stunnel-3.4a.orig/stunnel.c	Tue Jul 13 15:34:28 1999
+++ stunnel-3.4a/stunnel.c	Mon Sep 13 11:01:38 1999
@@ -41,7 +41,7 @@
 #else /* USE_WIN32 */
 
 /* directory for certificate */
-#define CERT_DIR ssldir "/certs"
+#define CERT_DIR "/var/state/openssl/certs"
 
 /* default certificate */
 #define DEFAULT_CERT CERT_DIR "/stunnel.pem"
