--- tools/Makefile.am~	2006-04-01 23:32:25.768966500 +0200
+++ tools/Makefile.am	2006-04-03 10:04:03.295469000 +0200
@@ -28,7 +28,7 @@
 		${INSTALL} -m 600 stunnel.pem $(DESTDIR)$(confdir)/stunnel.pem; \
 		rm stunnel.pem; \
 	fi
-	${INSTALL} -d -m 1770 -g nogroup $(DESTDIR)$(prefix)/var/lib/stunnel
+	${INSTALL} -d -m 1770 $(DESTDIR)$(prefix)/var/lib/stunnel
 	if uname | grep SunOS; then \
 		${INSTALL} -d -m 755 $(DESTDIR)$(prefix)/var/lib/stunnel/dev; \
 		mknod $(DESTDIR)$(prefix)/var/lib/stunnel/dev/zero c 13 12; \
