--- stunnel-3.20/configure.in.orig	Tue Oct 16 14:17:10 2001
+++ stunnel-3.20/configure.in	Tue Oct 16 14:20:01 2001
@@ -169,9 +169,9 @@
     LIBS="-lwrap $saved_LIBS"
     AC_TRY_LINK([int hosts_access(); int allow_severity, deny_severity;],
       [hosts_access()],
-      [AC_MSG_RESULT(yes); AC_DEFINE(HAVE_LIBWRAP)],
+      [AC_MSG_RESULT(yes); AC_DEFINE(HAVE_LIBWRAP) wrap_LIB="-lwrap"],
       [AC_MSG_RESULT(no); LIBS="$saved_LIBS"])
-       
+    LIBS="$saved_LIBS"
   ]
 )
 
