--- stunnel-5.49/src/client.c~	2018-08-26 19:36:09.000000000 +0200
+++ stunnel-5.49/src/client.c	2018-09-20 18:32:38.330919447 +0200
@@ -1202,7 +1202,8 @@ NOEXPORT void auth_user(CLI *c) {
     while(*user==' ') /* skip leading spaces */
         ++user;
     if(strcmp(user, c->opt->username)) {
-        s_log(LOG_WARNING, "Connection from %s REFUSED by IDENT (user \"%s\")",
+        s_log(LOG_WARNING, "Connection to service %s from %s REFUSED by IDENT (user \"%s\")",
+            c->opt->servname,
             c->accepted_address, user);
         str_free(line);
         throw_exception(c, 1);
