--- stunnel-3.9/Makefile.in~	Sun Dec 31 04:46:19 2000
+++ stunnel-3.9/Makefile.in	Sun Dec 31 04:53:44 2000
@@ -82,7 +82,7 @@
 $(OBJS): Makefile common.h
 
 stunnel.pem: stunnel.cnf
-	$(ssldir)/bin/openssl req -new -x509 -days 365 -nodes \
+	echo -e "\n\n\n\n\n\n" | $(ssldir)/bin/openssl req -new -x509 -days 365 -nodes \
 		-config stunnel.cnf -out stunnel.pem -keyout stunnel.pem
 	$(ssldir)/bin/openssl x509 -subject -dates -fingerprint -noout \
 		-in stunnel.pem
