--- stunnel-3.22/Makefile.in.orig	Mon Jan  7 14:16:47 2002
+++ stunnel-3.22/Makefile.in	Mon Jan  7 14:18:15 2002
@@ -103,7 +103,7 @@
 $(OBJS): Makefile $(HEADERS)
 
 stunnel.pem: stunnel.cnf
-	$(openssl) req -new -x509 -days 365 -nodes \
+	echo -e "\n\n\n\n\n\n" | $(openssl) req -new -x509 -days 365 -nodes \
 		-config stunnel.cnf -out stunnel.pem -keyout stunnel.pem
 	test $(NO_DH) -eq 1 || test ! -f $(RANDOM_FILE) || \
                 $(openssl) gendh -rand $(RANDOM_FILE) 512 >> stunnel.pem
