diff -burN stunnel-4.06.orig/configure.ac stunnel-4.06/configure.ac
--- stunnel-4.06.orig/configure.ac	2004-12-26 18:33:15.124066152 +0100
+++ stunnel-4.06/configure.ac	2004-12-26 18:34:28.262947352 +0100
@@ -33,6 +33,7 @@
 AC_CHECK_SIZEOF(unsigned short)
 AC_CHECK_SIZEOF(unsigned int)
 AC_CHECK_SIZEOF(unsigned long)
+AC_CHECK_SIZEOF(unsigned long long)
 
 AC_MSG_NOTICE([**************************************** PTY device files])
 AC_CHECK_FILE("/dev/ptmx", AC_DEFINE(HAVE_DEV_PTMX))
