--- stunnel-4.21/configure.ac.orig   2007-08-08 22:43:10.000000000 +0000
+++ stunnel-4.21/configure.ac        2007-10-28 11:44:32.424648485 +0000
@@ -92,8 +92,8 @@
 # Checks for standard libraries
 AC_SEARCH_LIBS(gethostbyname, nsl)
 AC_SEARCH_LIBS(yp_get_default_domain, nsl)
-AC_CHECK_LIB(socket, socket)
-AC_CHECK_LIB(util, openpty)
+AC_SEARCH_LIBS(socket, socket)
+AC_SEARCH_LIBS(openpty, util)
 # Checks for dynamic loader and zlib needed by OpenSSL
 AC_CHECK_LIB(dl, dlopen)
 AC_CHECK_LIB(dld, shl_load)
