--- stunnel-4.14/src/client.c.orig	2005-10-24 20:00:56.000000000 +0200
+++ stunnel-4.14/src/client.c	2006-01-24 13:31:39.000000000 +0100
@@ -735,8 +735,8 @@
     }
 
     if(!result) {
-        s_log(LOG_WARNING, "Connection from %s REFUSED by libwrap",
-            c->accepted_address);
+        s_log(LOG_WARNING, "Connection to service %s from %s REFUSED by libwrap",
+            c->opt->servname, c->accepted_address);
         s_log(LOG_DEBUG, "See hosts_access(5) manual for details");
         return -1; /* FAILED */
     }
