--- stunnel-5.49/tools/stunnel.conf-sample.in~	2018-04-06 16:25:10.000000000 +0200
+++ stunnel-5.49/tools/stunnel.conf-sample.in	2018-09-20 18:30:11.979864413 +0200
@@ -8,11 +8,11 @@
 ; **************************************************************************
 
 ; It is recommended to drop root privileges if stunnel is started by root
-;setuid = nobody
-;setgid = @DEFAULT_GROUP@
+setuid = stunnel
+setgid = stunnel
 
 ; PID file is created inside the chroot jail (if enabled)
-;pid = @localstatedir@/run/stunnel.pid
+pid = @localstatedir@/run/stunnel/stunnel.pid
 
 ; Debugging stuff (may be useful for troubleshooting)
 ;foreground = yes
