--- stunnel-3.22/Makefile.in.orig	Mon Jan  7 14:24:18 2002
+++ stunnel-3.22/Makefile.in	Mon Jan  7 14:30:24 2002
@@ -24,7 +24,7 @@
 LIBS=@LIBS@
 HEADERS=common.h prototypes.h client.h
 OBJS=client.o stunnel.o ssl.o protocol.o sthreads.o pty.o log.o options.o
-DESTFILES=$(sbindir)/stunnel $(libdir)/stunnel.so $(man8dir)/stunnel.8  $(PEM_DIR)/stunnel.pem
+DESTFILES=stunnel stunnel.so stunnel.8 stunnel.pem
 
 WINGCC=i386-mingw32msvc-gcc 
 WINCFLAGS=-O2 -Wall -DUSE_WIN32=1 -DHAVE_OPENSSL=1 -DFD_SETSIZE=4096 -DVERSION=\"@VERSION@\" -I../openssl-0.9.6b/outinc
@@ -38,6 +38,10 @@
 all: stunnel stunnel.8 stunnel.html stunnel.so stunnel.pem
 
 install: all installdirs $(DESTFILES)
+	$(INSTALL) stunnel $(DESTDIR)$(sbindir)
+	$(INSTALL) stunnel.so $(DESTDIR)$(libdir)
+	$(INSTALLMAN) stunnel.8 $(DESTDIR)$(man8dir)
+	$(INSTALL) stunnel.pem $(DESTDIR)$(PEM_DIR)
 
 uninstall:
 	rm -f $(DESTFILES)
@@ -63,8 +67,8 @@
 	gpg --yes --armor --detach-sign --force-v3-sigs ../dist/$(VERSION).tar.gz
 
 installdirs: mkinstalldirs
-	./mkinstalldirs $(sbindir) $(libdir) $(man8dir) $(PEM_DIR) $(piddir)
-	chmod a=rwx,+t $(piddir)
+	./mkinstalldirs $(DESTDIR)$(sbindir) $(DESTDIR)$(libdir) $(DESTDIR)$(man8dir) $(DESTDIR)$(PEM_DIR) $(DESTDIR)$(piddir)
+	chmod a=rwx,+t $(DESTDIR)$(piddir)
 
 # non-standard external rules
 
