--- stunnel-4.10/tools/Makefile.am.orig	2005-03-26 21:38:01.000000000 +0100
+++ stunnel-4.10/tools/Makefile.am	2005-05-11 13:30:00.000000000 +0200
@@ -20,7 +20,7 @@
 		else \
 			RND=""; \
 		fi; \
-		$(openssl) req -new -x509 -days 365 $$RND \
+		echo -e "\n\n\n\n\n\n" | $(openssl) req -new -x509 -days 365 $$RND \
 			-config $(srcdir)/stunnel.cnf \
 			-out stunnel.pem -keyout stunnel.pem; \
 		$(openssl) gendh $$RND 1024 >> stunnel.pem; \
