diff -durN stunnel-3.4a.orig/Makefile.in stunnel-3.4a/Makefile.in
--- stunnel-3.4a.orig/Makefile.in	Mon Jul 12 19:36:10 1999
+++ stunnel-3.4a/Makefile.in	Sun Sep 12 16:50:14 1999
@@ -2,12 +2,12 @@
 # Makefile for stunnel by Michal Trojnara 1998-1999 #
 #####################################################
 
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-sbindir=@sbindir@
-libdir=@libdir@
-mandir=@mandir@/man8
-ssldir=@ssldir@
+prefix=$(DESTDIR)@prefix@
+exec_prefix=$(DESTDIR)@exec_prefix@
+sbindir=$(DESTDIR)@sbindir@
+libdir=$(DESTDIR)@libdir@
+mandir=$(DESTDIR)@mandir@/man8
+ssldir=$(DESTDIR)@ssldir@
 certdir=$(ssldir)/certs
 @SET_MAKE@
 VERSION=stunnel-@VERSION@
