diff -ur stunnel-4.30/tools/stunnel.conf-sample.in stunnel-4.30.new/tools/stunnel.conf-sample.in
--- stunnel-4.30/tools/stunnel.conf-sample.in	2010-01-18 14:17:49.000000000 +0100
+++ stunnel-4.30.new/tools/stunnel.conf-sample.in	2010-01-21 15:17:41.173740613 +0100
@@ -4,19 +4,19 @@
 ; please read the manual and make sure you understand them
 
 ; certificate/key is needed in server mode and optional in client mode
-cert = @prefix@/etc/stunnel/mail.pem
-;key = @prefix@/etc/stunnel/mail.pem
+cert = /etc/stunnel/mail.pem
+;key = /etc/stunnel/mail.pem
 
 ; protocol version (all, SSLv2, SSLv3, TLSv1)
 sslVersion = SSLv3
 
 ; security enhancements for UNIX systems - comment them out on Win32
 ; for chroot a copy of some devices and files is needed within the jail
-chroot = @prefix@/var/lib/stunnel/
-setuid = nobody
-setgid = @DEFAULT_GROUP@
+;chroot = /var/lib/stunnel/
+setuid = stunnel
+setgid = stunnel
 ; PID is created inside the chroot jail
-pid = /stunnel.pid
+pid = /var/run/stunnel/stunnel.pid
 
 ; performance tunings
 socket = l:TCP_NODELAY=1
@@ -33,12 +33,12 @@
 ; CApath is located inside chroot jail
 ;CApath = /certs
 ; it's often easier to use CAfile
-;CAfile = @prefix@/etc/stunnel/certs.pem
+CAfile = /etc/stunnel/certs.pem
 ; don't forget to c_rehash CRLpath
 ; CRLpath is located inside chroot jail
 ;CRLpath = /crls
 ; alternatively CRLfile can be used
-;CRLfile = @prefix@/etc/stunnel/crls.pem
+CRLfile = /etc/stunnel/crls.pem
 
 ; debugging stuff (may useful for troubleshooting)
 ;debug = 7
@@ -49,17 +49,17 @@
 
 ; service-level configuration
 
-[pop3s]
-accept  = 995
-connect = 110
-
-[imaps]
-accept  = 993
-connect = 143
-
-[ssmtp]
-accept  = 465
-connect = 25
+;[pop3s]
+;accept  = 995
+;connect = 110
+
+;[imaps]
+;accept  = 993
+;connect = 143
+
+;[ssmtp]
+;accept  = 465
+;connect = 25
 
 ;[https]
 ;accept  = 443
